DataContent/ReadingDB/ProcessorDataService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using ExceptionsLogging;
+using ItemLibrary;
+using ItemLibrary.DataContexts;
+using Parsing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WebParser.ComponentsParser;
+
+namespace DataContent.ReadingDB.Services
+{
+    public class ProcessorDataService : IData<IEnumerable<Processor>>
+    {
+        private ComputerContext _db { get; set; }
```

Comment:
here we could implement Dependency Injection i guess. we should set DBContext via constructor and then use it. it would be something like:

private DbContext _db;

public ComputerDataService (DbContext db)
{
_db = db;
}

Reply:
Also tried, did not work